lib/main_common.pm

```diff
@@ -425,6 +425,9 @@ sub load_boot_tests {
         loadtest "boot/boot_from_pxe";
         set_var("DELAYED_START", get_var("PXEBOOT"));
     }
+    elsif (get_var("IPXE")) {
+        loadtest "installation/ipxe_install";
+    }
```

Comment:
So ```ipxe_install``` has never been scheduled before in ```main_common.pm``` ?

Reply:
No, it is usually loaded in job scheduler yaml files directly in test suites.